openassessment/templates/openassessmentblock/message/oa_message_incomplete.html

```diff
@@ -1,56 +1,54 @@
 {% load i18n %}
 {% spaceless %}
 <div id="openassessment__message__{{ xblock_id }}" class="openassessment__message message">
-    <h3 class="message__title">
-        {% trans "Assignment in progress" %}
-    </h3>
+    <h4 class="message__title">{% trans "Progress" %}</h4>
     <div class="message__content">
         <p>
             {% if training %}
                 {% if training_deadline_approaching %}
-                    {% trans "Learner training will close soon. Complete the learner training step to move on." %}
+                    {% trans "Assignment in progress. Learner training will close soon. Complete the learner training step to move on." %}
```

Comment:
For all messages in this section: "Assignment in progress." -> "This assignment is in progress."